frontend/pages/[project]/proposal/index.tsx

```diff
@@ -103,12 +106,17 @@ function ProposalForm() {
       body: JSON.stringify(parcel),
     };
 
-    //TODO add toast or something to indicate success/failure to client
-    var res = await fetch(url, init);
+    //TODO toast time out may cause issue if user hit back
```

Comment:
initially my plan was to do it this way and trying to have a global notification/toast component that we can re-use by other part of the app.

but later i changed my mind, because if the call fails for example like a typo reason, that the user can modify easaly, the form will still be there, but if we force back without the result of the call then the user has to repeate all the process allover again.